scootapi/server/job_status.go

```diff
@@ -33,6 +31,8 @@ func GetJobStatus(jobId string, sc s.SagaCoordinator) (*scoot.JobStatus, error)
 		js := scoot.NewJobStatus()
 		js.ID = jobId
 		js.Status = scoot.Status_NOT_STARTED
+		js.TaskStatus = make(map[string]scoot.Status)
+		js.TaskData = make(map[string]*scoot.RunStatus)
```

Comment:
Nm, this is a bigger change then. I think a StartSaga needs to be logged before a jobID is returned so I'm not sure why we're even hitting this block of code...